recipients-resolver/src/main/java/com/redhat/cloud/notifications/recipients/statup/StartupUtils.java

```diff
@@ -52,4 +69,52 @@ public String readGitProperties() {
             throw new UncheckedIOException(e);
         }
     }
+
+    public List<String> readKeystore(Optional<URI> keystoreFile, Optional<String> keystorePassword) {
+        List<String> result = new ArrayList<>();
+        String logMessage;
+        if (keystoreFile.isEmpty() || keystorePassword.isEmpty()) {
+            result.add("keystore file or password is empty");
+            return result;
+        }
+        try {
+            File f = new File(keystoreFile.get());
+            KeyStore ks = KeyStore.getInstance("JKS");
+            FileInputStream keystoreFileInputStream = new FileInputStream(f);
+            ks.load(keystoreFileInputStream, keystorePassword.get().toCharArray());
+
+            final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("EEE MMM dd HH:mm:ss z yyyy");
+            final ZonedDateTime currentUtcTime = ZonedDateTime.now(ZoneId.of("UTC"));
+
+            Iterator<String> aliasesIt = ks.aliases().asIterator();
+            while (aliasesIt.hasNext()) {
+                String alias = aliasesIt.next();
+                Date notAfter = ((X509Certificate) ks.getCertificate(alias)).getNotAfter();
+
+                ZonedDateTime utcExpDateTime = ZonedDateTime.ofInstant(notAfter.toInstant(), ZoneId.of("UTC"));
+
+                final String utcExpDateTimeStr = formatter.format(utcExpDateTime);
+
+                long diff = ChronoUnit.DAYS.between(currentUtcTime, utcExpDateTime);
+                if (diff < 10) {
+                    logMessage = String.format("Certificate '%s' is about to expire! (on %s)", alias, utcExpDateTimeStr);
+                    Log.fatalf(logMessage);
```

Comment:
Same for all logs entries:
```suggestion
                    Log.fatalf("Certificate '%s' is about to expire! (on %s)", alias, utcExpDateTimeStr);
```